### DISCUSSION TITLE
How build Docker Image from API Source Code?

### DISCUSSION THREAD

**Discussion by asemandeveloper** [langgenius/dify#10756] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

Please guide how to build docker image and container from api and sandbox source code. As in this [guide](https://docs.dify.ai/getting-started/install-self-hosted/start-the-frontend-docker-container), have you done the necessary guidance on creating an image from the web?

**Comment by crazywoola** [comment]
The best way to do it is look into the GitHub Actions. No fancy stuffs here. 
https://github.com/langgenius/dify/blob/6692e8c508f93c488f67b8ad0fedf5d914a86113/.github/workflows/build-push.yml#L1

However, if you changed the logo, you need to obtain a business license from us.

**Reply by asemandeveloper** [reply]
Unfortunately, your answer is not appropriate and has no solution. Please don't close my help request so that users who can help, give a proper answer or proper guidance.
